Tasks/DownloadPackageV1/multifilepackage.ts

```diff
@@ -74,7 +74,7 @@ export class MultiFilePackage extends Package {
                         if (filteredFileList.has(fileMetadatas[i].name)) {
                             const fileMetadata = fileMetadatas[i];
                             pkgFileUrlPromises.push(
-                                fileMetadata.protocolMetadata.data.StorageId != null
+                                fileMetadata.protocolMetadata.data.storageId != null
```

Comment:
Yay for JS being case sensitive

Reply:
and for onprem to have different casing than hosted :)

Reply:
we will have to make this fix before next onprem release goes out